plugins/Marketplace/stylesheets/marketplace.less

```diff
@@ -173,4 +173,30 @@
         margin-top: 20px;
         margin-bottom: 20px;
     }
-}
\ No newline at end of file
+}
+
+.manage-license-key {
+    .manage-license-key-input {
+        width: 50%;
```

Comment:
For other languages I guess this might even be the case for wider sizes, depending on the translation. And I also don't think it's worth the effort to handle such resolutions for that part "locally". Imho we would need some sort of global definition how to handle lower screen sizes, so it looks the same everywhere...